plugins/woocommerce/client/admin/client/wp-admin-scripts/coming-soon-newsletter-setting-panel/index.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { registerPlugin } from '@wordpress/plugins';
+import { useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { getAdminLink } from '@woocommerce/settings';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+declare global {
+	interface Window {
```

Comment:
What do you think if we update the global typings to include comingSoonNewsletter? I think this would make it easier to know what's available in the global scope.

https://github.com/woocommerce/woocommerce/blob/4607b5cb643c6596a78c6faf0749879b59395afa/plugins/woocommerce/client/admin/client/typings/global.d.ts#L1

Reply:
Updated it in [Move typing to typings/global.d.ts](https://github.com/woocommerce/woocommerce/pull/52650/commits/56850cf34ec3a86bf3b6904c5e2acdb9feddeb22) 🙏